pkg/operator/resource/resourceapply/apiextensions.go

```diff
@@ -55,3 +88,29 @@ func DeleteCustomResourceDefinitionV1(ctx context.Context, client apiextclientv1
 	resourcehelper.ReportDeleteEvent(recorder, required, err)
 	return nil, true, nil
 }
+
+// copyCustomResourceDefinitionConversionWebhookCABundle populates spec.conversion.webhook.clientConfig.caBundle fields from
+// existing resource if it was set before and is not set in present. This provides upgrade compatibility with service-ca-bundle operator.
+func copyCustomResourceDefinitionConversionWebhookCABundle(existing *apiextensionsv1.CustomResourceDefinition, required *apiextensionsv1.CustomResourceDefinition) {
```

Comment:
could you rename the params to `from` and `to` ? it would be easier to read the code.